clustering/jgroups/extension/src/main/resources/schema/jboss-as-jgroups_9_0.xsd

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
   ~ JBoss, Home of Professional Open Source.
-  ~ Copyright 2016, Red Hat, Inc., and individual contributors
+  ~ Copyright 2019, Red Hat, Inc., and individual contributors
```

Comment:
Let's do a 2022 here since this is publicly facing as to not cause confusion.

Reply:
Fixed